### PR TITLE
fix typo for dockerd.8.md

### PR DESCRIPTION
**Pull request by lixiaobing1**
fix typo.

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM 🐸
/cc @AkihiroSuda @runcom @thaJeztah

File: man/dockerd.8.md [hunk 1]
```diff
@@ -404,7 +404,7 @@ Managing the thin-pool outside of Engine makes for the most feature-rich
 method of having Docker utilize device mapper thin provisioning as the
 backing storage for Docker containers. The highlights of the lvm-based
 thin-pool management feature include: automatic or interactive thin-pool
-resize support, dynamically changing thin-pool features, automatic thinp
+resize support, dynamically changing thin-pool features, automatic thin
```
Comment (thaJeztah) [inline man/dockerd.8.md]: Are you sure it's a typo? I think `thinp` is the actual name here (for thin provisioning) https://fedoraproject.org/wiki/Features/ThinProvisioning

/cc @rhvgoyal

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
As @thaJeztah said this does not seem typo

**Comment by lixiaobing10051267** [comment]
ok

**Pull request closed without merging**